src/devices/Common/tests/GeographicPositionTest.cs

```diff
@@ -39,10 +40,31 @@ public void EqualityGeographicPosition()
         [Fact]
         public void DefaultFormatting()
         {
-            Assert.Equal("10° 00' 00.00\"N / 32° 30' 00.00\"E Ellipsoidal Height 12345m", new GeographicPosition(10.0, 32.5, 12345).ToString());
-            Assert.Equal("111° 06' 08.42\"S / 32° 59' 15.36\"W Ellipsoidal Height 200m", new GeographicPosition(-111.10234, -32.9876, 200).ToString());
-            Assert.Equal("11° 00' 00.00\"S / 33° 00' 00.00\"E Ellipsoidal Height 200m", new GeographicPosition(-10.99999999999999999, 32.999999999999999, 200).ToString());
+            Assert.Equal("10° 00' 00.00\"N 032° 30' 00.00\"E Ellipsoidal Height 12345m", new GeographicPosition(10.0, 32.5, 12345).ToString());
```

Comment:
0 in front of the numbers usually suggests octal system

Reply:
Not here. It is customary to always give all digits when writing or transmitting coordinates, as this prevents errors. Coordinates often need to be transmitted over poor radio channels.